api/users/serializers.py

```diff
@@ -66,7 +70,10 @@ def update(self, instance, validated_data):
                     instance.social[key] = val
             else:
                 setattr(instance, attr, value)
-        instance.save()
+        try:
+            instance.save()
+        except ValidationValueError as e:
+            raise InvalidModelValueError(detail=e.message)
```

Comment:
There don't appear to be any blank exceptions raised by the User model as far as I can tell, so this won't block review. But useful to know for the future.